c2corg_api/views/document_delete.py

```diff
@@ -289,6 +290,5 @@ def _remove_associations(document_id):
         )).delete()
 
 
-def _notify_es_syncer(config):
-    # TODO
-    pass
+def _notify_es_syncer(document_id):
+    DBSession.add(ESDeletedDocument(document_id=document_id))
```

Comment:
You will also have to send a message to Redis if the database transaction is successful. See the existing function  `notify_es_syncer`.